python/cudf/cudf/options.py

```diff
@@ -215,9 +241,28 @@ def _integer_and_none_validator(val):
         \tValid values are True or False. Default is False.
         """
     ),
-    _make_contains_validator([False, True]),
+    _spill_validator,
 )
 
+
+_register_option(
+    "copy_on_write",
+    _env_get_bool("CUDF_COPY_ON_WRITE", False),
+    textwrap.dedent(
+        """
+        Default behavior of performing shallow copies.
+        If set to `False`, each shallow copy will perform a true shallow copy.
+        If set to `True`, each shallow copy will perform a shallow copy
+        with underlying data actually referring to the actual column, in this
+        case a copy is only made when there is a write operation performed on
+        the column.
+        \tValid values are True or False. Default is False.
+    """
```

Comment:
Should we simply point to the `copy-on-write` docs here rather than trying to explain what copy on write means in a sentence?

Reply:
Yup, that makes sense. Updated to point to docs.